# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/ppc-select.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


We must not use the old select syscall ...

  - Rene Rebe <rene@exactcode.de>

--- /dev/null	2006-01-03 10:10:10.127999000 +0100
+++ dietlibc-0.29/ppc/select.S	2006-01-03 11:20:43.000000000 +0100
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(_newselect,select)
--- /dev/null	2006-01-03 10:10:10.127999000 +0100
+++ dietlibc-0.29/ppc64/select.S	2006-01-03 11:16:50.000000000 +0100
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(_newselect,select)
